js/wizard.js

```diff
@@ -281,51 +275,54 @@
 		},
 
 		previous: function () {
-			var canMovePrev = (this.currentStep > 1);
-			if( this.options.disablePreviousStep ) {
-				canMovePrev = false;
+			if( this.options.disablePreviousStep || this.currentStep === 1) {
+				return;
 			}
-			if (canMovePrev) {
-				var e = $.Event('actionclicked.fu.wizard');
-				this.$element.trigger(e, {step: this.currentStep, direction: 'previous'});
-				if (e.isDefaultPrevented()) { return; } // don't increment
 
-				this.currentStep -= 1;
-				this.setState();
-			}
+			var e = $.Event('actionclicked.fu.wizard');
+			this.$element.trigger(e, {step: this.currentStep, direction: 'previous'});
+			if (e.isDefaultPrevented()) { return; }// don't increment ...what? Why?
 
-			// return focus to control after selecting an option
-			if( this.$prevBtn.is(':disabled') ) {
-				this.$nextBtn.focus();
-			}
-			else {
-				this.$prevBtn.focus();
-			}
+			this.currentStep -= 1;
+			this.setState();
+
+			// only set focus if focus is still on the $nextBtn (avoid stomping on a focus set programmatically in actionclicked callback)
+			if( this.$prevBtn.is(":focus") ) {
+				var firstFormField = this.$element.find('.active').find('input, select, textarea')[0];
 
+				if(typeof firstFormField !== 'undefined'){
+					// allow user to start typing immediately instead of having to click on the form field.
+					$(firstFormField).focus();
+				}else if(this.$element.find('.active input:first').length === 0 && this.$prevBtn.is(':disabled') ) {
+					//only set focus on a button as the last resort if no form fields exist and the just clicked button is now disabled
+					this.$nextBtn.focus();
+				}
+			}
 		},
 
 		next: function () {
-			var canMoveNext = (this.currentStep + 1 <= this.numSteps);
-			var lastStep = (this.currentStep === this.numSteps);
-
-			if (canMoveNext) {
+			if (this.currentStep < this.numSteps) {
 				var e = $.Event('actionclicked.fu.wizard');
 				this.$element.trigger(e, {step: this.currentStep, direction: 'next'});
-				if (e.isDefaultPrevented()) { return; }	// don't increment
+				if (e.isDefaultPrevented()) { return; }// don't increment ...what? Why?
 
 				this.currentStep += 1;
 				this.setState();
-			}
-			else if (lastStep) {
+			} else {//is last step
 				this.$element.trigger('finished.fu.wizard');
 			}
 
-			// return focus to control after selecting an option
-			if( this.$nextBtn.is(':disabled') ) {
-				this.$prevBtn.focus();
-			}
-			else {
-				this.$nextBtn.focus();
+			// only set focus if focus is still on the $nextBtn (avoid stomping on a focus set programmatically in actionclicked callback)
+			if( this.$nextBtn.is(":focus") ) {
```

Comment:
This is minor and nit-picky, but could you change this line to ':focus' instead? It'd be more consistent with how strings are defined elsewhere in the file.